src/style.css

```diff
@@ -0,0 +1,252 @@
+* {
+  margin: 0;
+  border: 0;
+  font-family: Circular Std, sans-serif;
+  box-sizing: border-box;
+  color: #ebecf0;
+}
+
+body {
+  background: #1e2c36;
+}
+
+.main-header {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  gap: 20px;
+  margin: 5% 3%;
+}
+
+#logo {
+  width: 7%;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+  background-color: rgba(255, 217, 0, 0.167);
+}
+
+.header-link {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+  font-size: 20px;
+}
+
+.link {
+  width: auto;
+  padding: 5px;
+}
+
+.like-icon {
+  background-color: red;
+  border: 2px solid red;
+  width: 24px;
+  height: 24px;
+}
+
+.main-content {
+  margin: 5% 3% 5% 3%;
+  display: grid;
+
+  /* justify-content: center; */
```

Comment:
- [ ] To keep your project professional we strongly suggest you to remove all unused commented codes from your project. In style.css kindly remove all unused commented codes if they are not necessary to use.